### DISCUSSION TITLE
Passing parameters to x-bind element event does not work

### DISCUSSION THREAD

**Discussion by sivagurus** [alpinejs/alpine#3060] [1. Help] [answered]
Alpine.data('dropdown', () => ({
    open: false,
 
    trigger: {
        ['@click("hello")'](message) {
            console.log(message);
        },
    },
    dialogue: {
        ['x-show']() {
            return this.open
        },
    },
}))

In the above code not logging message instead of it returns $event.
How to pass parameters to x-bind trigger click event

**Comment by SimoTod** [comment, marked as answer]
You can't pass a parameter to a click function (a click is just a click).
